staging/vhost-device-sound/src/audio_backends/pipewire.rs

```diff
@@ -638,6 +643,8 @@ mod tests {
         let streams = Arc::new(RwLock::new(vec![Stream::default()]));
         let stream_params = streams.clone();
 
+        let _test_harness = PipewireTestHarness::new();
```

Comment:
I probably would have done something like:
```
PipewireTestHarness::session_scope(|| {
    ...
}
```
That avoids returning an object that has no real public methods to call on... But I got no strong opinions on this.

Reply:
The harness type specifically has public fields in case they are needed in the future:

```
    pub dbus: DbusSession,
    pub pipewire_child: Child,
    pub tempdir: TempDir,
```

and is marked `non_exhaustive` so it cannot be constructed out of its `new` method. There's no reason for a scope closure.